src/components/UISchema/index.tsx

```diff
@@ -146,6 +146,7 @@ class UISchema extends Component<Props, State> {
   validate = (callback: (error?: string) => void) => {
     this.form.validate((errors) => {
       if (errors) {
+        console.log(errors);
```

Comment:
Does the console .log function need to be removed?

Reply:
This is intentionally reserved, for quickly getting validatation messages.